packages/bundle/src/adaptiveCards/Attachment/CommonCard.js

```diff
@@ -32,13 +32,15 @@ export default class CommonCard extends React.Component {
     return (
       <AdaptiveCardRenderer
         adaptiveCard={ content && this.buildCard(adaptiveCards, content) }
-        tapAction={ tap }
+        adaptiveCardHostConfig={ adaptiveCardHostConfig }
+        tapAction={ content && tap }
```

Comment:
Okay to keep it or not.

Currently, if `content` is falsy, `tap` will always be `undefined`. No need to worry about null reference exception here.

But your extra logic could also means, "no matter what changes in the future, if there is no content, the user cannot tap on it."

Reply:
this change was because of an eslint error. it says 'tap' is never used. Do you have a different recommendation?

Reply:
![image](https://user-images.githubusercontent.com/1622400/57735865-e0133480-765a-11e9-8479-96bb4b0ef0ac.png)

It looks good on eslint.